content/patterns/multicloud-gitops-amx-rhoai/mcg-amx-rhoai-demo-script.adoc

```diff
@@ -0,0 +1,156 @@
+
+[id="demo-objectives"]
+
+== Objectives
+
+In this demo you will complete the following:
+
+* Prepare your local workstation
+* Deploy the pattern
+* Extend the pattern with a small tweak
+
+[id="getting-started"]
+
+== Getting Started
+
+* Make sure you have met all the link:/learn/quickstart/#installation_prerequisitesrequirements[requirements]
+* Follow the link:../mcg-amx-thoai-getting-started[Getting Started Guide] to ensure that you have met all of the prequisites 
+
+[NOTE]
+====
+This demo begins after `./pattern.sh make install` has been executed
+====
+
+[id="demo"]
+
+== Demo
+
+Now that we have deployed the pattern onto our cluster, with `origin` pointing to your fork and using `my-branch` as the name of the used branch, we can begin to discover what has happened.
+You should be able to click on the nine-box and see the following entries:
+
+image:multicloud-gitops/nine-box.png[]
+
+If you now click on the "Hub ArgoCD" menu entry you will be taken to the ArgoCD instance with all the applications.
+
+image:multicloud-gitops/hub-argocd.png[]
+
+
+
+[id="secrets"]
+
+=== Secrets loading
+
+==== With pattern.sh script
+
+By default in the MultiCloud GitOps pattern the secrets get loaded automatically via an out of band process inside the vault running in the OCP cluster. This means that running `./pattern.sh make install` will also call the `load-secrets` makefile target.
+This `load-secrets` target will look for a yaml file describing the secrets to be loaded into vault and in case it cannot find one it will use the `values-secret.yaml.template` file in the git repo to try and generate random secrets.
+
+Let's copy the template to our home folder and reload the secrets:
+[source,terminal]
+cp ./values-secret.yaml.template ~/values-secret-multicloud-gitops.yaml
+./pattern.sh make load-secrets
+
+At this point if the `config-demo` application was not green already it should become green in the ArgoCD user interface.
+
+==== Manually
+
+Another way to load secrets is to add them for `config-demo` application (from _values-secret-multicloud-gitops.yaml_) to *Vault* manually:
+
+. Go to Vault service route. URL can be found:
+
+.. by running command:
++
+[source,terminal]
+----
+oc -n vault get route vault -ojsonpath='{.spec.host}'
+----
++
+.. in *Openshift Container Platform* web console under *Networking > Routes* for `vault` project.
+
+. Log into the Vault using root token. Root token can be found by executing command:
++
+[source,terminal]
+----
+oc -n imperative get secrets vaultkeys -ojsonpath='{.data.vault_data_json}' | base64 -d
+----
++
+
+. After login go to `secret` catalog and clik *Create secret* and fill all the fields manually (Figure 2):
+
+.. *Path for this secret* is `global/config-demo` (from _values.yaml_ file for `config-demo` charts)
+
+.. Under *Secret data* key is `secret` (from _values-secret-multicloud-gitops.yaml_ file) and in next field put its value.
+
+.. Click *Add* and then *Save*.
```

Comment:
One more comment...

TBH I don't get it... Let me try to rephrase it, and decide if this is clear:

a) Put `global/config-demo` value in the _Path for this secret_ field (the value cames from _values-secret-multicloud-gitops.yaml_ file).
b) Add one _Secret data_ key-value pair. Put `secret` as a key (left field). The value must acquired from _values-secret-multicloud-gitops.yaml_ file. Click _Add_ button to confirm.
c) Click _Save_ to save changes.

THIS IS HOW I UNDERSTAND THIS PART - PLEASE VALIDATE A LOGIC BEFORE REPLACING YOUR PART ;)

**BTW is this section twice? In the "Getting started" page, and in the "Demo Script"?**